src/components/utils/index.js

```diff
@@ -74,14 +74,26 @@ export const selectBorderStyle = type => {
 }
 
 export const createSocialMetas = metas => [
-  <meta property="og:title" content={metas.title} />,
-  <meta property="og:description" content={metas.description} />,
-  <meta property="og:type" content="article" />,
-  <meta property="og:image" content={metas.image} />,
-  <meta name="twitter:site" content="@reactgqlacademy" />,
-  <meta name="twitter:title" content={metas.title} />,
-  <meta name="twitter:description" content={metas.description} />,
-  <meta name="twitter:creator" content="@reactgqlacademy" />,
+  <meta key={'og:title'} property="og:title" content={metas.title} />,
```

Comment:
I guess this was just to solve that key error when we map :-)

Reply:
yep, there was a warning in dev because of this